### PR TITLE
Twitter training data

### PR DESCRIPTION
**Pull request by mguida22**
Uses a training set built up of tagged tweets. A few things of importance:

- 800,000 tagged tweets (half positive, half negative) Cut the data to reasonable size for testing using the `trim.py` script. You can specify a threshold within there.
- We're at ~55% accuracy using this data with the current setup, however the `word_feats` function is currently **not working** as intended so there may be a significant increase by fixing that.
- To help with manual testing of phrases there's a tiny cli script, just run and enter any string to test.

Speed has been a slight concern, but doesn't look like it'll be a problem. A full run with 800,000 tweets takes around 5 minutes to run on my computer (this will vary, but is a good ballpark).

screen shot 2016-04-06 at 8 00 11 pm

cc @FredLoh

### CODE REVIEW

**Comment by FredLoh** [comment]
@mguida22 could you send me your trained classifier .pickle file?  Guess python2 vs 3 makes a huge difference.

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,4 @@ coverage.xml
 *.pickle
 
 sentiment/config.json
+*.csv
```
Comment (719Ben) [inline .gitignore]: Why add `.csv`?

**Comment by 719Ben** [comment]
@mguida22 I know we talked about moving this over to Go. Should we keep it Python or move to Go?

File: .gitignore (hunk 1 above)
Comment (mguida22) [inline .gitignore]: Doesn't need to be a catch all, but we have some large csv files that don't really make sense to commit - we only use them for training of the sentiment classifier.

**Comment by mguida22** [comment]
@719Ben as far as Python vs Go I think we should leave this in Python because it works and is fast enough to retrain. Actually classifying once we have a trained model is a trivial amount of time. I still think it'd be cool to have the webserver in Go. Maybe a pipeline would work like... storm -> python classifier -> storm -> go webserver -> client. Idk if thats a smart way or not
